corehq/toggles.py

```diff
@@ -306,3 +306,8 @@ def toggles_dict(username=None, domain=None):
     'add_help_text',
     "Adds a help text in the form builder"
 )
+
+VELLUM_TRANSACTION_QUESTION_TYPES = StaticToggle(
+    'transaction_question_types',
+    "Adds transaction-related question types in the form builder"
+)
```

Comment:
Should this be able to be enabled by domain? Not sure what will happen when two users who edit the form when one can see them and the other can't